test/GCClassic/integration/README.md

```diff
@@ -29,6 +29,8 @@ Integration tests are short GEOS-Chem Classic simulations.  Executables are comp
 
 - `commonFunctionsForTests.sh`
   - Link to `../../shared/commonFunctionsForTests.sh`, which contains global variables and functions for the integration and parallelization tests.
+- `integrationDiffTest.sh`
```

Comment:
If we have this instead in test/diffTest then it could have its own README on how to use it. Does it need to be in integration?

Reply:
We could move it out @lizziel.